services/management/management.go

```diff
@@ -213,32 +318,37 @@ func (s *service) initMetrics(metricFactory metric.Factory) {
 	s.metrics.currentTopology = metricFactory.NewText("Management.Topology.Current")
 }
 
-func (s *service) updateMetrics() {
+func (s *service) updateMetrics(isSuccessful bool) {
 	s.RLock()
 	defer s.RUnlock()
 
 	currentRef := s.data.CurrentReference
 
 	s.metrics.currentRefTime.Update(int64(currentRef))
 	s.metrics.genesisRefTime.Update(int64(s.data.GenesisReference))
-	s.metrics.lasUpdateTime.Update(time.Now().Unix())
+	s.metrics.pageStartRefTime.Update(int64(s.data.StartPageReference))
+	s.metrics.pageEndRefTime.Update(int64(s.data.EndPageReference))
+	s.metrics.lastUpdateTime.Update(time.Now().Unix())
+	if isSuccessful {
```

Comment:
why not count the errors (in an else clause)?

Reply:
i don't care how many errors were. we really just need to know when is the last successfull the actual number is irrelevant.

Reply:
it would be good for maintenance / debugging